### PR TITLE
Add Audit Logs Suite

### PR DESCRIPTION
**Pull request by sheldonvaughn**

### CODE REVIEW

File: tests/WorkOS/AuditLogsTest.php [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+        return [
+        "success" => true
+    ];
+    }
+
+    private function createEventResponseFixture()
+    {
+        return json_encode([
+        "success" => true
+    ]);
```
Comment (jonatascastro12) [inline tests/WorkOS/AuditLogsTest.php:143]: nit: Could we fix the indentation across this file?

```suggestion
        return json_encode([
            "success" => true
        ]);
```

File: tests/WorkOS/AuditLogsTest.php [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    {
+        return [
+        "object" => "audit_log_export",
+        "id" => "audit_log_export_123",
+        "state" => "ready",
+        "url" => "https://audit-logs.com/download.csv",
+        "createdAt" => "2022-08-18T18:07:10.822Z",
+        "updatedAt" => "2022-08-18T18:07:10.822Z",
+    ];
+    }
```
Comment (jonatascastro12) [inline tests/WorkOS/AuditLogsTest.php:156]: other example:

```suggestion
    private function createExportFixture()
    {
        return [
            "object" => "audit_log_export",
            "id" => "audit_log_export_123",
            "state" => "ready",
           "url" => "https://audit-logs.com/download.csv",
           "createdAt" => "2022-08-18T18:07:10.822Z",
           "updatedAt" => "2022-08-18T18:07:10.822Z",
       ];
    }
```

Comment (jonatascastro12) [reply tests/WorkOS/AuditLogsTest.php:156]: tip: https://stackoverflow.com/a/29973358

File: lib/AuditLogs.php [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        $createExportPath = "audit_logs/exports";
+
+        $params = [
+         "actions" => $actions,
+         "actors" => $actors,
+         "organization_id" => $organizationId,
+         "range_end" => $rangeEnd,
+         "range_start" => $rangeStart,
+         "targets" => $targets
+        ];
```
[the remark this answers is not in the document]
Comment (sheldonvaughn) [reply lib/AuditLogs.php]: Added logic to accomplish this - let me know if there's a more elegant solution!

File: tests/WorkOS/AuditLogsTest.php (hunk 1 above)
Comment (sheldonvaughn) [reply tests/WorkOS/AuditLogsTest.php]: Formatted these!

File: tests/WorkOS/AuditLogsTest.php (hunk 2 above)
Comment (sheldonvaughn) [reply tests/WorkOS/AuditLogsTest.php]: Formatted here as well

File: lib/AuditLogs.php [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+        $createExportPath = "audit_logs/exports";
+
+        $params = [
+         "organization_id" => $organizationId,
+         "range_end" => $rangeEnd,
+         "range_start" => $rangeStart
+        ];
+
+        if (!is_null($actions)) {
+            $params += ["actions" => $actions];
```
Comment (jonatascastro12) [inline lib/AuditLogs.php:82]:
```suggestion
            $params["actions"] = $actions;
```

File: lib/AuditLogs.php [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+        };
+
+        if (!is_null($actors)) {
+            $params += ["actors" => $actors];
```
Comment (jonatascastro12) [inline lib/AuditLogs.php:86]:
```suggestion
            $params["actors"] = $actors;
```

File: lib/AuditLogs.php [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+        };
+
+        if (!is_null($targets)) {
+            $params += ["targets" => $targets];
```
Comment (jonatascastro12) [inline lib/AuditLogs.php:90]:
```suggestion
            $params["targets"] = $targets;
```

**Review by jonatascastro12 — APPROVED** [review]
LGTM.
nit: Left an even simpler suggestion.

**Pull request merged**